### PR TITLE
[CONTENT] Ledger Integration english content

### PR DESCRIPTION
**Pull request by gantunesr**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**

Add english content for the Ledger integration.

**Screenshots/Recordings**

~_If applicable, add screenshots and/or recordings to visualize the before and after of your change_~

**Issue**

Progresses https://github.com/MetaMask/mobile-planning/issues/637

**Checklist**

* [x] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

**Review by montelaidev — APPROVED** [review]
LGTM

**Review by blackdevelopa — APPROVED** [review]

File: locales/languages/en.json [hunk 1]
```diff
@@ -2324,5 +2324,67 @@
...
+    "ledger_disconnected_error": "The connection to your device has been lost. Please try again.",
+    "unknown_error": "Unexpected error occurred.",
+    "unknown_error_message": "An unexpected error occured. Please try again.",
+    "error_occured": "An error occured",
+    "how_to_install_eth_app": "How to install the Ethereum app on a Ledger device",
+    "ledger_account_count": "You are using 1 account from your Ledger with MetaMask Mobile.",
+    "open_eth_app": "Please open the Ethereum app",
+    "open_eth_app_message_one": "We’ve detected you have the Ethereum app installed but it’s not open, please ",
+    "open_eth_app_message_two": "press the two buttons on the device to accept the prompt to open the Ethereum app to continue.",
+    "toast_bluetooth_connection_error_title": "Oops, something went wrong :/",
```
Comment (NicolasMassart) [inline locales/languages/en.json:2377]: I noticed that we already replaced the "oops" by something better in other prompts to make them more actionable.

File: locales/languages/en.json [hunk 2]
```diff
@@ -2324,5 +2324,67 @@
...
+    "toast_bluetooth_connection_error_subtitle": "Ledger has not been connected",
```
Comment (NicolasMassart) [inline locales/languages/en.json:2378]:
```suggestion
    "toast_bluetooth_connection_error_subtitle": "Can't connect to Ledger",
```

File: locales/languages/en.json [hunk 3]
```diff
@@ -2324,5 +2324,67 @@
...
+    "try_again": "Try again",
+    "forget_device": "Forget Ledger",
+    "sign_with_ledger": "Sign with Ledger",
+    "ledger_pending_confirmation": "Ledger is busy",
+    "ledger_pending_confirmation_error": "There is a pending action on your Ledger. Please clear the action first then retry.",
+    "not_supported": "Operation not supported",
+    "not_supported_error": "Only version 4 of typed data signing is supported.",
+    "error_during_connection": "An unknown error occured",
+    "error_during_connection_message": "There's been a slight problem connecting your Ledger Nano X, tap 'Retry' below to give this another go. Sometimes this occurs due to the ETH app on your Ledger Nano X being open at the start of the pairing process with MetaMask Mobile.",
```
Comment (NicolasMassart) [inline locales/languages/en.json:2387]:
```suggestion
    "error_during_connection_message": "Unable to connect your Ledger Nano X, tap 'Retry'. This issue can be caused by the ETH app on your Ledger Nano X being open at the start of the pairing process with MetaMask Mobile.",
```

File: locales/languages/en.json [hunk 4]
```diff
@@ -2324,5 +2324,67 @@
       "wrong_answer_title": "Nope! Never share your Secret Recovery Phrase with anyone, ever",
       "wrong_answer_description": "Anyone claiming to need your Secret Recovery Phrase is lying to you. If you share it with them, they will steal your assets."
     }
+  },
+  "ledger": {
+    "open_settings": "Open settings",
+    "view_settings": "View settings",
+    "bluetooth_off": "Bluetooth is off",
+    "bluetooth_off_message": "Please turn on Bluetooth for your device",
```
Comment (NicolasMassart) [inline locales/languages/en.json:2332]: What device? Phone or Ledger?

File: locales/languages/en.json [hunk 5]
```diff
@@ -2324,5 +2324,67 @@
...
+    "bluetooth_access_blocked": "Ledger needs Bluetooth access to pair with your mobile device.",
+    "bluetooth_access_blocked_message": "If you want to pair your Ledger device using Bluetooth, enable it in Settings & try again.",
```
Comment (NicolasMassart) [inline locales/languages/en.json:2334]: What settings? MM or Ledger?

File: locales/languages/en.json [hunk 6]
```diff
@@ -2324,5 +2324,67 @@
...
+    "location_access_blocked": "Ledger needs location access to pair with your mobile device.",
+    "location_access_blocked_message": "If you want to pair your Ledger device using Bluetooth, you’ll need to enable location access in Settings & try again.",
```
Comment (NicolasMassart) [inline locales/languages/en.json:2336]: I know there's no location settings in Ledger, but for clarity and consistency, maybe indicate "mobile device settings" instead.

File: locales/languages/en.json [hunk 7]
```diff
@@ -2324,5 +2324,67 @@
...
+    "bluetooth_scanning_error": "Error while scanning the device",
+    "bluetooth_scanning_error_message": "Please make sure your device is unlocked and the Ethereum app is running",
```
Comment (NicolasMassart) [inline locales/languages/en.json:2338]: What device is unlocked? Ledger or mobile?
Would be good to have consistency on this like in L2340

File: locales/languages/en.json [hunk 8]
```diff
@@ -2324,5 +2324,67 @@
...
+    "bluetooth_connection_failed": "Bluetooth connection failed",
+    "bluetooth_connection_failed_message": "Please make sure your Ledger is unlocked and your Bluetooth is enabled",
+    "ethereum_app_open": "Ledger requires confirmation",
+    "ethereum_app_open_message": "Please confirm on your device to open Ethereum app. Press OK once done.",
```
Comment (NicolasMassart) [inline locales/languages/en.json:2342]: Same, what device? Apply this comment on all sentences where device is mentioned.

File: locales/languages/en.json [hunk 9]
```diff
@@ -2324,5 +2324,67 @@
...
+    "ethereum_app_unconfirmed_error": "You have denied the request to open the Ethereum app.",
+    "failed_to_open_eth_app": "Failed to open the Ethereum app.",
+    "ethereum_app_open_error": "Please install/accept Ethereum app on your Ledger device.",
```
Comment (NicolasMassart) [inline locales/languages/en.json:2345]: What does accept exactly mean here? Once app installed on device, you can open it or not, but I can't see what accept is.

File: locales/languages/en.json [hunk 10]
```diff
@@ -2324,5 +2324,67 @@
...
+    "running_app_close": "Failed to close the running app.",
+    "running_app_close_error": "Failed to close the running app on your Ledger device.",
+    "ethereum_app_not_installed": "Ethereum app not installed.",
+    "ethereum_app_not_installed_error": "Please install the Ethereum app on your Ledger device.",
+    "ledger_is_locked": "Ledger is locked",
+    "unlock_ledger_message": "Please unlock your Ledger device",
+    "cannot_get_account": "Cannot get account",
+    "connect_ledger": "Connect Ledger",
+    "looking_for_device": "Looking for device",
+    "ledger_reminder_message": "Please make sure your Ledger Nano X is:",
```
Comment (NicolasMassart) [inline locales/languages/en.json:2355]: We say Ledger all along, and here it's Ledger Nano X... what if we support other models later?

File: locales/languages/en.json [hunk 11]
```diff
@@ -2324,5 +2324,67 @@
...
+    "ledger_reminder_message_step_one": "1. Unlocked",
+    "ledger_reminder_message_step_two": "2. The Ethereum app is installed",
+    "ledger_reminder_message_step_three": "3. Bluetooth is enabled",
+    "ledger_reminder_message_step_four": "4. Location is enabled",
+    "available_devices": "Available devices",
+    "retry": "Retry",
+    "continue": "Continue",
+    "confirm_transaction_on_ledger": "Confirm transaction by pressing two buttons on your Ledger",
```
Comment (NicolasMassart) [inline locales/languages/en.json:2363]:
```suggestion
    "confirm_transaction_on_ledger": "Confirm transaction by pressing both buttons on your Ledger",
```

File: locales/languages/en.json [hunk 12]
```diff
@@ -2324,5 +2324,67 @@
...
+    "bluetooth_enabled_message": "Make sure Bluetooth is enabled",
+    "device_unlocked_message": "Device is unlocked",
+    "blind_signing_message": "Blind signing is enabled",
+    "ledger_disconnected": "Your device got disconnected",
```
Comment (NicolasMassart) [inline locales/languages/en.json:2367]:
```suggestion
    "ledger_disconnected": "Your device is disconnected",
```

File: locales/languages/en.json [hunk 13]
```diff
@@ -2324,5 +2324,67 @@
...
+    "ledger_disconnected_error": "The connection to your device has been lost. Please try again.",
+    "unknown_error": "Unexpected error occurred.",
+    "unknown_error_message": "An unexpected error occured. Please try again.",
```
Comment (NicolasMassart) [inline locales/languages/en.json:2370]: Error are most of the time unexpected...
```suggestion
    "unknown_error_message": "An unknown error occured. Please try again.",
```
But I would even fusion this with the error_occured one... knowing it's unexpected or unknown doesn't add anything ot the user understanding. It just makes them more anxious...

File: locales/languages/en.json [hunk 14]
```diff
@@ -2324,5 +2324,67 @@
...
+    "open_eth_app": "Please open the Ethereum app",
+    "open_eth_app_message_one": "We’ve detected you have the Ethereum app installed but it’s not open, please ",
+    "open_eth_app_message_two": "press the two buttons on the device to accept the prompt to open the Ethereum app to continue.",
+    "toast_bluetooth_connection_error_title": "Oops, something went wrong :/",
+    "toast_bluetooth_connection_error_subtitle": "Ledger has not been connected",
+    "try_again": "Try again",
+    "forget_device": "Forget Ledger",
+    "sign_with_ledger": "Sign with Ledger",
+    "ledger_pending_confirmation": "Ledger is busy",
+    "ledger_pending_confirmation_error": "There is a pending action on your Ledger. Please clear the action first then retry.",
```
Comment (NicolasMassart) [inline locales/languages/en.json:2383]:
```suggestion
    "ledger_pending_confirmation_error": "An action is in progress on your Ledger. Please finish or cancel the action, then retry.",
```

**Review by NicolasMassart — COMMENTED** [review]
Some comments on text style and complexity

**Pull request merged**